rllib/evaluation/env_runner_v2.py

```diff
@@ -815,7 +819,13 @@ def _handle_done_episode(
                         env_id,
                         agent_id,
                         {
-                            SampleBatch.T: new_episode.length - 1,
+                            # Pass dummy data into this together with obs since view
```

Comment:
@gjoliver Is this the proper way to initialize the view requirements connector with a first observation?

Reply:
you should not need these dummy data.
the first time VR connector is called, it will use add_init_obs() which doesn't expect all the other data fields.
are you seeing errors that made you do this?
if so, maybe you can rebase to Master. I fixed an issue around it yesterday.

Reply:
Yeah that's what I thought :/ 

This "processed = policy.agent_connectors(acd_list)" call is inside _handle_done_episode -> collector is not empty. So we either have to clear the buffers or add dummy values so satisfy the requirements, right?